src/transformers/results.js

```diff
@@ -1,4 +1,4 @@
-export const transformResults = ({ node }) => ({
+const transformResults = ({ node }) => ({
```

Comment:
Porque foi alterado para CommonJS, sendo que o resto dos módulos está usando ESM?
```suggestion
export const transformResults = ({ node }) => ({
```

Reply:
Porque agora ele é usado apenas no `gatsby-node` e lá só funciona commonjs porque roda no node e tal.

Reply:
talvez até faça sentido mover essa func direto pro gatsby-node uma vez que só está sendo usada lá também :thinking:

Reply:
Faz muito mais sentido, ótimo ponto!